README.md

```diff
@@ -54,7 +57,7 @@ Using Ruby...
 Write your code here...
 
 ```ruby
-# code here
+town[:residents].delete("Belle")
 ```
```

Comment:
To complete the answer, add Belle to the castle guests: town[:castle][:guests] << "Belle"